types/autobahn-browser/index.d.ts

```diff
@@ -0,0 +1,299 @@
+// Type definitions for AutobahnJS Browser Build 18.10
+// Project: https://crossbar.io/autobahn/, https://github.com/crossbario/autobahn-js-browser
+// Definitions by: Elad Zelingher <https://github.com/darkl>, Andy Hawkins <https://github.com/a904guy>, Wladimir Totino <https://github.com/valepu>, Mathias Teier <https://github.com/glenroy37>, Fran Rodriguez <https://github.com/spcfran>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="when" />
+
+export = autobahn;
```

Comment:
You don't want `export =` here instead you want define them in global since these would need to be available only in global.
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
I'm sorry but I don't understand this.